modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java

```diff
@@ -186,6 +186,21 @@
             + " WHERE " + QUEUE_ID + "=?"
             + " AND " + DLC_QUEUE_ID + "=-1";
 
+    protected static final String ALIAS_FOR_QUEUES = "QUEUE_COUNT";
+
+    /**
+     * Prepared statement to select all the queue names with the number of messages remaining in the database
+     * by joining the tables MB_QUEUE_MAPPING and MB_METADATA.
+     */
+    protected static final String PS_SELECT_ALL_QUEUE_MESSAGE_COUNT =
+            "SELECT " + QUEUE_NAME + ", " + PS_ALIAS_FOR_COUNT
+            + " FROM " + QUEUES_TABLE + " LEFT OUTER JOIN "
```

Comment:
Yeah, it is supported in mysql, mssql and oracle. When I went through reviews for H2, some say it isn't working properly. But I did test it with H2 and it had no issue.